src/display-buffer.coffee

```diff
@@ -692,6 +692,7 @@ class DisplayBuffer extends Model
     targetLeft = pixelPosition.left
     defaultCharWidth = @defaultCharWidth
     row = Math.floor(targetTop / @getLineHeightInPixels())
+    targetLeft = -Infinity if row < 0
```

Comment:
I'm not sure. I guess that would be OK, but as I don't really understand the internals of display-buffer (and fact that Infinity is used for the positive overflow case) I think Infinity might make more sense. Maybe it doesn't work this way, but I can imagine cases where there's some negative text indent that causes lines of text to layout in negative x territory, and so that's why Infinity seems a bit safer.